.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+Note: this section will not show up in the issue.
+Have you search for this feature before requesting it? It's highly likely that a similar request was already filed.
+-->
+
+## Describe your environment
+(if applicable)
+
+  * Operating system: ____
+  * Python Version: _____ (`python -V`)
+  * CCXT version: _____ (`pip freeze | grep ccxt`)
+  * Version: ____ (`freqtrade -V` | `docker-compose run --rm freqtrade -V`)
```

Comment:
```suggestion
  * Freqtrade Version: ____ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
```

(the pipe `|` can be recognized as the part of the command line, combining both these commands)
same in other places

Reply:
so can the or ...

Reply:
yes, but you know that pipes are parts of shell commands, one could simply copy-paste the whole "command" inside the parenthesis...